pages/in/widget/iscn-ar/index.vue

```diff
@@ -266,7 +266,8 @@ export default {
       title: '',
       type: 'article',
       author: '',
-      description: '',
+      authorDescription: '',
+      ISCNDescription: '',
```

Comment:
I think `description` for ISCNDescription is ok, since all these field should be related to `ISCN` if not otherwise specified